src/libsystemd/sd-netlink/netlink-types.c

```diff
@@ -701,6 +791,16 @@ uint16_t type_system_get_count(const NLTypeSystem *type_system) {
         return type_system->count;
 }
 
+void type_system_get_root(int protocol, const NLTypeSystem **root) {
```

Comment:
would't it make more sense to change this to:

`const NLTypeSystem* type_system_get_root(int protocol)`?